Packs/Carbon_Black_Enterprise_Response/.pack-ignore

```diff
@@ -17,19 +17,32 @@ ignore=IN126
 ignore=RM102,RM104
 
 [file:CBLiveGetFile_V2.yml]
-ignore=BA108,BA109,BA124
+ignore=BA108,BA109,BA124,GR107
 
 [known_words]
 cb
 watchlists
-PREPREPRE
-POSTPOSTPOST
+preprepre
+postpostpost
```

Comment:
intentional lowercasing?

Reply:
I don't remember this changes, i will revert this